src/lib/components/auth-cards/CreateCard.svelte

```diff
@@ -273,6 +297,11 @@
     height: var(--formHeight);
   }
 
+  .invalid-email{
+    margin-top: -19px;
```

Comment:
i’m cool with this, but i’m curious if we have sizing units/variables that are used throughout?

Reply:
Yes, we do but since this style is specific for this card, I didn't use a variable.